snapcraft/tests/commands/test_snap.py

```diff
@@ -305,6 +308,10 @@ def test_snap_containerized_inject_snap_dangerous(self,
             call(['lxc', 'file', 'push',
                   os.path.join(rundir, 'core_123.snap'),
                   '{}/run/core_123.snap'.format(container_name)]),
+            call(['sudo', 'cp', '/var/lib/snapd/snaps/snapcraft_x1.snap',
```

Comment:
Maybe a logger entry with the command to call `sudo` with here would be good (just so it is no surprise to the random user that decided to install our snap dangerously :wink:)